src/TRestAxionTrueWolterOptics.cxx

```diff
@@ -321,12 +333,12 @@ Int_t TRestAxionTrueWolterOptics::SecondMirrorReflection(const TVector3& pos, co
     TVector3 vertex(0, 0, fBackVertex[mirror]);
     Double_t focal = fR3[mirror] / TMath::Tan(4 * fAlpha[mirror]);
```

Comment:
Is `fR3` the middle point along the *extensions of mirrors 1 and 2* /or/ the virtual reflection point? If it is the former it's not the right equation, if `focal` is supposed to be the focal length of the telescope. 

(this is how you end up with a "1500mm" LLNL telescope, which actually has a 1530mm focal length)

Or is this equation for some reason correct as is for a true Wolter optic?

Reply:
So in the document for the true Wolter optics they call this the z-Koordinate of the detector/ focal point. I thought this would be roughly the same for each layer. What do you mean by "virtual reflection point"?

Reply:
So for these true Wolter functions it is not the focal length of the whole optics but of the the single hyperbolic mirror as far as I understand.

Reply:
![llnl_layers_explanation](https://github.com/rest-for-physics/axionlib/assets/7742232/33e6c738-d07e-4f65-bf0d-6c74b0105c50)

here as an SVG so that it's visible embedded.

Reply:
> So for these true Wolter functions it is not the focal length of the whole optics but of the the single hyperbolic mirror as far as I understand.

That makes some sense. But I would still be wary if the correct radius in that case corresponds to R3 or an equivalent of the virtual reflection point I indicate.

Reply:
[spiegel_parameter.pdf](https://github.com/rest-for-physics/axionlib/files/14375514/spiegel_parameter.pdf)

Here is the true Wolter calculation I got from Vadim's colleague and it does state that what I call "focal" is calculated with R3. Interesting point though to note about the virtual reflection point!

Reply:
Ok, good to know. Then I _suppose_ this is indeed correct for a true Wolter optic.

Reply:
But it's not the whole optics focal spot, that is true and confusing^^

Reply:
But the PDF you sent explicitly says 'Fokallänge' for `x_0 - x_f`?